tagstudio/src/qt/pagination.py

```diff
@@ -181,201 +187,22 @@ def update_buttons(self, page_count: int, index: int, emit: bool = True):
-                    )
-                else:
-                    self.start_ellipses.setMinimumWidth(self.button_size.width())
-                    self.start_ellipses.setMaximumWidth(self.button_size.width())
-            elif page_count == 11:
-                if index == 0:
-                    self.end_ellipses.setMinimumWidth(self.button_size.width() * 5 + 12)
-                    self.end_ellipses.setMaximumWidth(self.button_size.width() * 5 + 12)
-                elif index == 1:
-                    self.end_ellipses.setMinimumWidth(self.button_size.width() * 4 + 9)
-                    self.end_ellipses.setMaximumWidth(self.button_size.width() * 4 + 9)
-                elif index == 2:
-                    self.end_ellipses.setMinimumWidth(self.button_size.width() * 3 + 6)
-                    self.end_ellipses.setMaximumWidth(self.button_size.width() * 3 + 6)
-                elif index == 3:
-                    self.end_ellipses.setMinimumWidth(self.button_size.width() * 2 + 3)
-                    self.end_ellipses.setMaximumWidth(self.button_size.width() * 2 + 3)
-                else:
-                    self.end_ellipses.setMinimumWidth(self.button_size.width())
-                    self.end_ellipses.setMaximumWidth(self.button_size.width())
-                if index == page_count - 1:
-                    self.start_ellipses.setMinimumWidth(
-                        self.button_size.width() * 5 + 12
-                    )
-                    self.start_ellipses.setMaximumWidth(
-                        self.button_size.width() * 5 + 12
-                    )
-                elif index == page_count - 2:
-                    self.start_ellipses.setMinimumWidth(
-                        self.button_size.width() * 4 + 9
-                    )
-                    self.start_ellipses.setMaximumWidth(
-                        self.button_size.width() * 4 + 9
-                    )
-                elif index == page_count - 3:
-                    self.start_ellipses.setMinimumWidth(
-                        self.button_size.width() * 3 + 6
-                    )
-                    self.start_ellipses.setMaximumWidth(
-                        self.button_size.width() * 3 + 6
-                    )
-                elif index == page_count - 4:
-                    self.start_ellipses.setMinimumWidth(
-                        self.button_size.width() * 2 + 3
-                    )
-                    self.start_ellipses.setMaximumWidth(
-                        self.button_size.width() * 2 + 3
-                    )
-                else:
-                    self.start_ellipses.setMinimumWidth(self.button_size.width())
-                    self.start_ellipses.setMaximumWidth(self.button_size.width())
+            # I do not know where these magic values were derived from, but
+            # this is better than the chain elif's that were here before
+            if page_count >= 8 and page_count <= 11:
```

Comment:
```suggestion
            if 8 <= page_count <= 11:
```

Reply:
Thanks! I forget python has that, implemented.